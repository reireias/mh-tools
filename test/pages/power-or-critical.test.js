import { mount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/pages/power-or-critical'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('power-or-critical', () => {
  it('should be vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = mount(target, {
      localVue,
      vuetify
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
