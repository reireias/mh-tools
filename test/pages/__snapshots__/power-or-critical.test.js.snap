// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`power-or-critical should be vue instance 1`] = `
"<div>
  <div class=\\"title-text primary--text\\">MHW IB - 攻撃 or 見切り</div>
  <div class=\\"description-text\\">
    スキル「攻撃」と「見切り」をどれくらい積むか悩んだときに用の期待値計算機です。
    <br>
    物理ダメージのみに対応しています。
  </div>
  <div class=\\"v-item-group theme--light v-expansion-panels v-expansion-panels--popout\\">
    <div aria-expanded=\\"false\\" class=\\"v-expansion-panel\\"><button type=\\"button\\" class=\\"v-expansion-panel-header\\">基礎攻撃力の計算方法<div class=\\"v-expansion-panel-header__icon\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-chevron-down theme--light\\"></i></div></button>
      <div class=\\"v-expansion-panel-content\\" style=\\"display: none;\\"></div>
    </div>
  </div>
  <div class=\\"container\\">
    <div class=\\"row\\">
      <div class=\\"col\\">
        <div class=\\"v-input number-input v-input--is-label-active v-input--is-dirty theme--light v-text-field\\">
          <div class=\\"v-input__control\\">
            <div class=\\"v-input__slot\\">
              <div class=\\"v-text-field__slot\\"><label for=\\"input-11\\" class=\\"v-label v-label--active theme--light\\" style=\\"left: 0px; position: absolute;\\">基礎攻撃力</label><input id=\\"input-11\\" type=\\"number\\"></div>
            </div>
            <div class=\\"v-text-field__details\\">
              <div class=\\"v-messages theme--light\\">
                <div class=\\"v-messages__wrapper\\"></div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class=\\"col\\">
        <div class=\\"v-input number-input v-input--is-label-active v-input--is-dirty theme--light v-text-field\\">
          <div class=\\"v-input__control\\">
            <div class=\\"v-input__slot\\">
              <div class=\\"v-text-field__slot\\"><label for=\\"input-14\\" class=\\"v-label v-label--active theme--light\\" style=\\"left: 0px; position: absolute;\\">会心率</label><input id=\\"input-14\\" type=\\"number\\">
                <div class=\\"v-text-field__suffix\\">%</div>
              </div>
            </div>
            <div class=\\"v-text-field__details\\">
              <div class=\\"v-messages theme--light\\">
                <div class=\\"v-messages__wrapper\\"></div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div class=\\"col\\">
        <div class=\\"v-input v-input--is-label-active v-input--is-dirty theme--light v-text-field v-select\\">
          <div class=\\"v-input__control\\">
            <div role=\\"button\\" aria-haspopup=\\"listbox\\" aria-expanded=\\"false\\" aria-owns=\\"list-17\\" class=\\"v-input__slot\\">
              <div class=\\"v-select__slot\\">
                <div class=\\"v-select__selections\\">
                  <div class=\\"v-select__selection v-select__selection--comma\\">超会心Lv0</div><input id=\\"input-17\\" readonly=\\"readonly\\" type=\\"text\\" aria-readonly=\\"true\\">
                </div>
                <div class=\\"v-input__append-inner\\">
                  <div class=\\"v-input__icon v-input__icon--append\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu-down theme--light\\"></i></div>
                </div>
              </div>
              <div class=\\"v-menu\\">
                <div class=\\"v-menu__content theme--light \\" style=\\"max-height: 304px; min-width: 0px; max-width: auto; top: 12px; left: 0px; transform-origin: top left; z-index: 0; display: none;\\"></div>
              </div>
            </div>
            <div class=\\"v-text-field__details\\">
              <div class=\\"v-messages theme--light\\">
                <div class=\\"v-messages__wrapper\\"></div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <center><button type=\\"button\\" class=\\"v-btn v-btn--contained theme--light v-size--default primary\\"><span class=\\"v-btn__content\\">計算</span></button></center>
  <div class=\\"result-box\\">
    <center>
      <!---->
    </center>
  </div>
</div>"
`;
